india_compliance/gst_india/doctype/gst_settings/gst_settings.json

```diff
@@ -379,12 +381,27 @@
    "fieldname": "enable_e_waybill_from_pr",
    "fieldtype": "Check",
    "label": "Enable e-Waybill Generation from Purchase Receipt"
+  },
+  {
+   "default": "1",
```

Comment:
One setting for e-Invoice & e-Waybill and one Cron Job. In API Section. Conditional display